docs/commerce/discounts/work_with_discounts.md

```diff
@@ -0,0 +1,154 @@
+Cart discount applicability can be further limited by setting a number of conditions, such as:
+
+- a number of products in the cart
+- total purchase value
+- a discount code
+
+!!! note "Navigating through the steps"
+
+    When you define discount details, you can go back to change your choices.
+    To do it, click a step header at the top of the screen.
+
+    ![Discount creator steps](discount_creator_headers.png)
+
+1. In the left panel, go to **Commerce** -> **Discounts**, and click **Create**.
+
+2. Select whether the discount applies to catalog or cart products and the product's type.
+
+    Choose **Fixed amount** to deduct a specific amount of money from the base price of the product, or **Percentage** to calculate the deducted amount based on a specific percent value.
+
+    ![New discount](new_discount.png)
+
+3. Provide general information about the new discount:
+
+    1. In the **Global properties** area, provide an internal name of the discount and set the validity period.
+    Toggle the **Permanent discount** on to make the discount valid until you manually disable it.
+    1. Then, select discount [priority]([[= developer_doc =]]/discounts/discounts_guide/#discounts-priority) to help the system choose the right discount to apply when calculating the final price.
+    1. If your store supports multiple markets, you can select a region that the discount applies to, and decide whether the discount applies to all or selected currencies.
+    1. In the **Promotion label** field, provide a name of the campaign, as it should be shown to customers, for example "Summer Sale" or "DACH Conference".
+    1. Click **Next** to go to the next screen.
+    
+    ![Creating a new discount](create_new_discount.png)
+
+4. Select customers that the new discount is targeted at.
+You can choose everyone, or select one or more customer groups.
+
+    ![Customer selection](discounts_select_customers.png)
+
+5. Select products that the discount applies to. You can choose between:
+    
+    - all products from the catalog, for example, to clear stock before the end of year
+    - products from a specific category, for example, promotional gadgets for company partners
+    - specific products or even product variants, to promote a specific brand
+
+    In the latter case, you select products by using a Product picker, where you can use search and filters to pinpoint the exact product or product variant that you want the discount to apply to.
+
+    ![Product picker](product_picker.png)
+
+6. If you are creating a cart discount, in the next you screen, you can set the conditions that limit the discount's availability to customers who have:
+
+    - added to cart no less than a specific number of certain items
+    - added products to a cart for no less than a specified total value
+    - entered a specified discount code
+
+    If you set the discount code, you can also set the number of times that the code can be used.
```

Comment:
This might need some more clarification and screenshot update since global discount codes are now part of the feature.